canonical/lxd-ui
# chore: fix app wide accessibility issues

## Done

- disable link in logo for small screen (reason: same adjacent link URL)
- changed title attribute so it's not identical to text content (InstanceList)
- TablePagination aria-label
- associate orphaned form label (InstanceCreateDetailsForm)
- add text to hidden input
- Resolve multiple label for the same form control (SnapshotScheduleInput)
- Remove uneccessary element titles or adjust for better meaning
- Adjusted heading elements to not skip heirarchical order
- Added aria-label for Select components that's missing labels

In addition, here is the [Jira ticket](https://warthogs.atlassian.net/browse/WD-8559) that documents all open a11y issues.

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described [here](../HACKING.md#setting-up-for-development).
2. Perform the following QA steps:
    - Download Wave evaluation tool, check the application and see if there are any errors/alerts present on different pages and not documented as an open issue on this `[Jira ticket](https://warthogs.atlassian.net/browse/WD-8559)`.
